test/ads.jl

```diff
@@ -0,0 +1,129 @@
+using TaylorIntegration
+using Test
+
+@testset "Automatic Domain Splitting" begin
```

Comment:
I'd suggest to add a test where we know there will be no splitting (a linear harmonic oscillator for example) and see that we don't produce splitting there. Also, a test involving the RTBP for unbounded orbits (aka CR3BP) can be interesting.

Reply:
A possibility is to check that the first few steps run without any splitting